### PR TITLE
added commits

### PR DESCRIPTION
**Pull request by Sarthak5598**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -15,6 +15,10 @@ gitpython = "^3.1.43"
 slack-sdk = "^3.27.2"
 slackeventsapi = "^3.0.1"
 requests = "^2.32.3"
+pytest = "^8.2.2"
+pytest-mock = "^3.14.0"
+pytest-asyncio = "^0.23.7"
+ruff = "^0.4.8"
```
Comment (arkid15r) [inline pyproject.toml:21]: These should be in `test` dependency group so you'd be able to use `poetry install --with test`

**Review by arkid15r — CHANGES_REQUESTED** [review]
Please make sure the tests are running and don't fail

**Comment by Sarthak5598** [comment]
image
tests are fine now ,also corrected the ci-cd file

**Review by arkid15r — APPROVED** [review]
Looks good 👍

**Comment by Sarthak5598** [comment]
Thank you!

**Pull request merged**